### DISCUSSION TITLE
Hosting: System/Hardware requirements (e.g. for hosting in EC2)

### DISCUSSION THREAD

**Discussion by mnordhaus** [umputun/remark42#1876] [Q&A] [answered]
Hi all,

Of course this question depends on how much traffic your website gets. But I am still interested in how to you host Remark42 and what to expect when it comes to dimensioning the hosting environment.

I am leaning towards an EC2 instance and wonder what kind of configuration I would need for a small personal blog (like 500 to 1000 visitors a month). But I also like to hear from others with maybe larger environments.

I guess that could help a lot of people and might be worse to document. So help from everyone is much appreciated. 

And thank you for creating such a cool open source project!

Best,

Markus

**Comment by paskal** [comment]
Smallest possible instance should work with that load. I'm yet to see big enough load to learn the bottlenecks, I wonder if anyone with big traffic can post their `docker stats` output.

**Reply by mnordhaus** [reply]
Thank you @paskal I'll give that a try and see how it scales.

**Comment by paskal** [comment, marked as answer]
I've published the system requirements based on ~five years old instance to https://remark42.com/docs/getting-started/installation/.
